ci/common.rb

```diff
@@ -42,6 +42,26 @@ def travis_pr?
   !ENV['TRAVIS'].nil? && ENV['TRAVIS_EVENT_TYPE'] == 'pull_request'
 end
 
+BAD_CITIZENS = {
+  'couch' => 'couchdb',
+  'disk' => 'sysstat',
```

Comment:
I agree on the overhead of adding a new flavor, we should avoid that. Not sure about adding it to `core_integration` though since the check is not really part of the core of the Agent (and we always make the distinction between core of the agent and checks in our test flavors)

Another possibility is to consolidate under the same flavor all the check integration tests that don't need any specific install steps. AFAIK that would include the `disk`, `sysstat`, `network` and `docker_daemon` tests. This could have the drawback of adding confusion to the flavor names though (especially for the `docker_daemon` one, it's not intuitive to have it grouped with these other tests)

OR, we could just keep the `sysstat` flavor as it is but add the `network` check test to it, so that the `sysstat` flavor aggregates all the tests on the "system" checks that don't need any install. Thoughts?